src/westpa/core/_rc.py

```diff
@@ -322,6 +322,11 @@ def new_sim_manager(self):
             from .binning.mab_manager import MABSimManager
 
             sim_manager = MABSimManager(rc=self)
+        elif binmapname == 'BinlessMapper':
+            sim_manager = westpa.core.binning.binless_manager.BinlessManager(rc=self)
+            sys.stderr.write("-- WARNING  [config] -- Do not use BinlessMapper as an outer binning scheme with a target state.\n")
+        elif nestmapname == 'BinlessMapper':
```

Comment:
Same as above. Depending on where these variables are from, I wonder if there is a better way of checking these conditions. It may have to use string names in the end but string names can be very unreliable.